rest-api/alembic/versions/be736e26410b_add_primary_language_to_participant_.py

```diff
@@ -0,0 +1,58 @@
+"""add_primary_language_to_participant_summary
+
+Revision ID: be736e26410b
+Revises: 378ee128c23f
+Create Date: 2018-10-16 14:51:38.736406
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from participant_enums import MetricSetType, MetricsKey
+from model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = 'be736e26410b'
+down_revision = '378ee128c23f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('primary_language', sa.String(length=80), nullable=True))
```

Comment:
It may be better to do these in one Alembic migration. Probably doesn't matter though.

Reply:
Do you mean submit the Alembic change in a separate branch?

Reply:
generally that is considered a best practice, also, we can probably remove the original languageId from the original alembic file instead of creating it and then deleting it everytime. Maybe Dan can comment on that.

Reply:
We should not remove the language ID from the original alembic file; if we want to drop the column we want that applied to existing databases.

We certainly can merge together these two migrations; delete  both of them, reset your database, and run generate_schema again, and Alembic should do the drop and create together.